force-app/main/default/classes/PS_Request.cls

```diff
@@ -74,6 +75,7 @@ public inherited sharing class PS_Request {
     public static String ENDPOINT_PURCHASE = '/v1/payments/verified/purchase';
     public static String ENDPOINT_COMMITMENT = '/v1/payments/verified/commitments';
     public static String ENDPOINT_COMMITMENT_CANCEL = '/v1/payments/verified/commitments/{0}/cancel';
+    public static String ENDPOINT_COMMITMENT_UPDATE_BULK = '/v1/payments/internal/commitments/update/bulk';
```

Comment:
Using Internal endpoint for now. Will switch to Verified endpoint once it's completed.